test/gtest/ucs/test_lru.cc

```diff
@@ -24,25 +24,30 @@ class test_lru : public ucs::test {
         ucs::test::cleanup();
     }
 
+    static ucs_status_t lru_get(void *item, void *opaque)
+    {
+        std::vector<void*> *elems = (std::vector<void*>*)opaque;

```

Comment:
1. auto
2. use retinterpret_cast
3. maybe define a type for this vector

Reply:
function removed